src/providers/index.js

```diff
@@ -9,6 +9,7 @@ import Bungie from './bungie'
 import Cognito from './cognito'
 import Credentials from './credentials'
 import Discord from './discord'
+import Dropbox from './dropbox'
```

Comment:
Could you revert this file change to get rid of the merge conflict? This file is now autogenerated!